aspnetcore/blazor/handle-errors.md

```diff
@@ -0,0 +1,214 @@
+---
+title: Handle errors in ASP.NET Core Blazor apps
+author: guardrex
+description: Discover how ASP.NET Core Blazor how Blazor manages unhandled exceptions and how to develop apps to detect and handle errors.
+monikerRange: '>= aspnetcore-3.0'
+ms.author: riande
+ms.custom: mvc
+ms.date: 07/20/2019
+uid: blazor/handle-errors
+---
+# Handle errors in ASP.NET Core Blazor apps
+
+By [Steve Sanderson](https://github.com/SteveSandersonMS) and [Luke Latham](https://github.com/guardrex)
+
+This article describes how Blazor manages unhandled exceptions and how to develop apps to detect and handle errors.
```

Comment:
```suggestion
This article describes how Blazor manages unhandled exceptions and how to develop apps that detect and handle errors.
```

Suggested phrasing is because people won't build an app whose primary purpose is to detect and handle errors. Rather, that's one of the secondary concerns of app building.